remoulade/brokers/rabbitmq.py

```diff
@@ -184,11 +199,14 @@ def _declare_dq_queue(self, channel, queue_name):
         return channel.queue.declare(queue=dq_name(queue_name), durable=True, arguments=arguments)
 
     def _declare_xq_queue(self, channel, queue_name):
-        return channel.queue.declare(queue=xq_name(queue_name), durable=True, arguments={
+        arguments = {
             # This HAS to be a static value since messages are expired
             # in order inside of RabbitMQ (head-first).
             "x-message-ttl": DEAD_MESSAGE_TTL,
-        })
+        }
+        if self.dead_queue_max_length:
```

Comment:
If you set a `dead_queue_max_length` to 0, it won't be enforced

Reply:
hum ok. If think 0 has no sense so i'll forbid it.